# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
    
FLS_Joint_position_controller:
  type: position_controllers/JointPositionController
  joint: FLS_Joint
  pid: {p: 100.0, i: 1.0, d: 10.0}
  
FRS_Joint_position_controller:
  type: position_controllers/JointPositionController
  joint: FRS_Joint
  pid: {p: 100.0, i: 1.0, d: 10.0}
    
BLS_Joint_position_controller:
  type: position_controllers/JointPositionController
  joint: BLS_Joint
  pid: {p: 100.0, i: 1.0, d: 10.0}  

BRS_Joint_position_controller:
  type: position_controllers/JointPositionController
  joint: BRS_Joint
  pid: {p: 100.0, i: 1.0, d: 10.0}  
